tags/crosspost.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Please don't crosspost your support questions!
+color: orange
+---
+
+Please be patient and wait for somebody to give support. Reposting your question in multiple channels can be disruptive both for yourself and any others in the channel. This also doesn't get you support quicker.
```

Comment:
```suggestion
Please be patient and wait for somebody to give support. Reposting your question in multiple channels can be disruptive for others in the channel. This also doesn't get you support quicker.
```
"Disruptive for yourself" doesn't really make sense

Reply:
eh I see wym, I had an intention when writing it but I forgot now